x-pack/plugins/infra/server/lib/adapters/nodes/lib/create_partition_bodies.ts

```diff
@@ -24,6 +25,9 @@ export function createPartitionBodies(
   const { sourceConfiguration }: InfraNodeRequestOptions = nodeOptions;
   const bodies: InfraESMSearchBody[] = [];
   const numberOfPartitions: number = Math.ceil(totalNodes / NODE_REQUEST_PARTITION_SIZE);
+  const indices = nodeOptions.metrics.every(m => m.type === InfraMetricType.logRate)
+    ? [sourceConfiguration.logAlias]
+    : [sourceConfiguration.logAlias, sourceConfiguration.metricAlias];
```

Comment:
What happens if the graphql request contains both a `logRate` metric and something like a `cpu` metric? From what I can observe, this leads to incorrect log rate calculations because it counts the docs from metric indices as well. :thinking: